packages/components/src/card/docs/Card.stories.mdx

```diff
@@ -52,13 +52,13 @@ A card must have an heading and a content.
     </Story>
 </Preview>
 
-### Outline
+### Leveled
```

Comment:
Brought the topic in slack, but i'm copying it here. I'm not sure about renaming variants like this. It might lead to an API that isn't constant. We have outline selects, outline tags, outline cards. If we only rename some, it might lead to inconsistencies. Changes should mostly be CSS, not variant names